bench/wtperf/runners/chunk-cache-reads-base.wtperf

```diff
@@ -0,0 +1,10 @@
+conn_config="statistics=(all),statistics_log=(json=true,on_close=true,wait=1),cache_size=50MB,eviction=(threads_max=8)"
+readonly=false
+table_count=5
+icount=25000000
+key_sz=1024
+value_sz=1024
+threads=((count=10,reads=1))
+run_time=120
```

Comment:
Just wondering if you have tried running it for longer, for instance, ~ 15 minutes? Was the performance difference similar?

Reply:
Good question - I tried 5 earlier and it was about the same. No harm in trying 15, I'll kick that off now.

Reply:
OK I tried 15 and it was still the same number of reads/minute.

Reply:
Thanks